apps/showcase/src/components/utilities/date-picker-input-hebrew/date-picker-input-pres.config.ts

```diff
@@ -0,0 +1,8 @@
+import type {Configuration} from '@o3r/core';
+import {computeItemIdentifier} from '@o3r/core';
+
+export interface DatePickerInputPresConfig extends Configuration {}
```

Comment:
We can keep the component simplier if config is not needed

Reply:
Initially the type was needed for c11n directive. I'll double check if I could not pass the Configuration interface directly and remove this interface.